mantle/cosa/cosa_v1.go

```diff
@@ -36,6 +36,7 @@ type Build struct {
 	ContainerConfigGit        *Git                  `json:"coreos-assembler.container-config-git,omitempty"`
 	CoreOsSource              string                `json:"coreos-assembler.code-source,omitempty"`
 	CosaContainerImageGit     *Git                  `json:"coreos-assembler.container-image-git,omitempty"`
+	CosaDelayedMetaMerge      bool                  `json:"coreos-assembler.delayed-meta-merge,omitempty"`
```

Comment:
> coroes-assembler.delayed-meta-merge: boolean for determining when and if meta.json should be written to artifacted paths.

WDYT about using a term like "sub-build"?  Or alternatively we're already using the term "extend" in "buildextend" so maybe "extend-build"?  Something like one of those feels more self-descriptive than "delayed meta merge" or "artifacted path".

- coreos-assembler.subbuild
- coreos-assember.buildextension

?

Reply:
But eh this was specified in the enhancement and I didn't comment then so